http/src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/20210414_add_home_dir.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="leonardo" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <changeSet logicalFilePath="leonardo" author="qi" id="add_home_directory">
+        <addColumn tableName="CLUSTER_IMAGE">
+            <column name="homeDirectory" type="varchar(254)">
+                <constraints nullable="true"/>
+            </column>
+        </addColumn>
+
+        <sql stripComments="true">
+            update CLUSTER_IMAGE set homeDirectory = "/home/jupyter-user" where imageType = "Jupyter";
```

Comment:
wonder if it's worth adding for rstudio runtimes as well in case we need it in the future? would just be `/home/rstudio`

Reply:
I can add it in a follow-up PR....

Reply:
we could also just create a ticket to do it in the future since i can't think of an immediate need for it now. after reading the rest of the PR it's not as straight forward to add as i initially thought